examples/chacha20/proofs/fstar/extraction/Chacha20.fst

```diff
@@ -21,15 +21,17 @@ let chacha20_line (a b d: usize) (s: u32) (m: t_Array u32 (sz 16))
       (fun _ -> Prims.l_True) =
   let state:t_Array u32 (sz 16) = m in
   let state:t_Array u32 (sz 16) =
-    Rust_primitives.Hax.update_at state
+    Rust_primitives.Hax.Monomorphized_update_at.update_at_usize state
       a
       (Core.Num.impl__u32__wrapping_add (state.[ a ] <: u32) (state.[ b ] <: u32) <: u32)
   in
   let state:t_Array u32 (sz 16) =
-    Rust_primitives.Hax.update_at state d ((state.[ d ] <: u32) ^. (state.[ a ] <: u32) <: u32)
+    Rust_primitives.Hax.Monomorphized_update_at.update_at_usize state
+      d
+      ((state.[ d ] <: u32) ^. (state.[ a ] <: u32) <: u32)
   in
   let state:t_Array u32 (sz 16) =
-    Rust_primitives.Hax.update_at state
+    Rust_primitives.Hax.Monomorphized_update_at.update_at_usize state
```

Comment:
Was this code out of date?

Reply:
Yes, the CI just discard the F* snapshot, so there is no policy regarding how up-to-date things are